src/react-components/CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 ========
 
 ## [0.3.6]
-- added `Template` stories
+
```

Comment:
Can we review when the changelog should be bumped vs. just adding the update to the current changelog number?

Reply:
It should be bumped after a release.  Ideally we'd have releases at set times, (sprint end) but ... that's probably not going to happen for now.  

I think the clearest thing to do here is to pre-bump the changelog as a part of release, so that they never need to be bumped (unless there's a breaking change). I'll make a ticket to put that in the readme.